include/alp/views.hpp

```diff
@@ -66,6 +68,18 @@ namespace alp {
 			}
 		};
 
+		template< typename OriginalType >
+		struct Gather {
+
+			using applied_to = OriginalType;
+
+			static constexpr Views type_id = Views::gather;
+
+			static std::pair< size_t, size_t > dims( std::pair< size_t, size_t > dims_pair ) {
```

Comment:
I think we can drop this as it is no longer required by the AMF-based infrastructure.